core/domain/role_services.py

```diff
@@ -33,12 +33,16 @@
 ACTION_ACCESS_LEARNER_DASHBOARD = 'ACCESS_LEARNER_DASHBOARD'
 ACTION_ACCESS_MODERATOR_PAGE = 'ACCESS_MODERATOR_PAGE'
 ACTION_ACCESS_TOPICS_AND_SKILLS_DASHBOARD = 'ACCESS_TOPICS_AND_SKILLS_DASHBOARD'
+ACTION_CHANGE_QUESTION_STATUS = 'CHANGE_QUESTION_STATUS'
+ACTION_CHANGE_TOPIC_STATUS = 'CHANGE_TOPIC_STATUS'
 ACTION_CREATE_COLLECTION = 'CREATE_COLLECTION'
 ACTION_CREATE_EXPLORATION = 'CREATE_EXPLORATION'
+ACTION_CREATE_QUESTION = 'CREATE_NEW_QUESTION'
```

Comment:
Probably should order alphabetically, ditto below.

Reply:
Done.